### PR TITLE
Add Rails

### PR DESCRIPTION
**Pull request by denny**

### CODE REVIEW

File: config/master.key [hunk 1]
```diff
@@ -0,0 +1 @@
+d1399141ddd73ff3e240dce1c035ebc3
```
Comment (eliotsykes) [inline config/master.key]: This file looks sensitive and that it shouldn't be tracked by git, especially not in a public repo. 

Double-check this is correct, and if so, treat any credentials encrypted in `config/credentials.yml.enc` as exposed and invalidate/reset them. Also generate a fresh `master.key`, remove it from git permanently, and re-encrypt `config/credentials.yml.enc`.

Comment (denny) [reply config/master.key]: D'oh! Thanks.

Comment (denny) [reply config/master.key]: I didn't know either of these files existed, so I don't think there's anything of consequence in them?

Comment (eliotsykes) [reply config/master.key]: Try decrypting it to check:
https://github.com/rails/rails/blob/master/railties/lib/rails/commands/credentials/USAGE

If there's any secrets in there, such as the one used to encrypt cookies, then that will need regenerating. Also regenerate `master.key` as its exposed, even if nothing sensitive is in `credentials.yml.enc` yet.

Comment (eliotsykes) [reply config/master.key]: Try running: `EDITOR="atom --wait" bin/rails credentials:edit`

Comment (denny) [reply config/master.key]: Ah yes, the cookie key is in it (nothing else though). Thanks again!

Comment (denny) [reply config/master.key]: I've gone for the brute force approach of removing the files, and adding SECRET_BASE_KEY to .env instead

**Pull request merged**